### ISSUE TITLE
Blank interactive section

### ISSUE THREAD

**Issue by ChaoticDreamer** [Zulu-Inuoe/git-go#9]
A fun image or summary about what's about to happen could be added to the interactive slide.

**Comment by Zulu-Inuoe** [comment]
Yeah that's probably a good idea. Let me see if I can figure out how to make a checklist, that way I can tick each topic as we cover them.
